docs/src/cookbooks/cookbook.md

```diff
@@ -1,10 +1,14 @@
 ---
 layout: docs
-title:  "Cookbook"
+title:  "General Cookbooks"
```

Comment:
```suggestion
title:  "General Cookbook"
```

Reply:
changing this to match the below... if I need to fix the menu.yml in the other PR I will make it "General Cookbook"